pkg/services/reputation/local/route/calls.go

```diff
@@ -0,0 +1,136 @@
+package reputationroute
+
+import (
+	"sync"
+
+	"github.com/nspcc-dev/neofs-node/pkg/services/reputation"
+	reputationcontroller "github.com/nspcc-dev/neofs-node/pkg/services/reputation/local/controller"
+	"go.uber.org/zap"
+)
+
+type routeContext struct {
+	reputationcontroller.Context
+
+	passedRoute []ServerInfo
+}
+
+func (r *routeContext) Epoch() uint64 {
```

Comment:
Method is redundant, it is implemented through inheritance

Reply:
forget to delete after previous changes. deleted